docker-cloud/slack-integration.md

```diff
@@ -4,14 +4,9 @@ aliases:
 description: Integrate with Slack
 keywords:
 - Slack, integrate, notifications
-menu:
-  main:
-    parent: docker-cloud
-title: Docker Cloud notifications in Slack
+title: Receive Docker Cloud notifications in Slack
```

Comment:
This is probably more like "configure" or "set up", since receiving is sort of a passive thing that only happens once you've done that bit...